# HG changeset patch
# Parent fe43a939d102bf70879c57198c692e0a3278f4ad
# Enabling UCLIBC_HAS_BACKTRACE for puma models

diff --git a/toolchain/uClibc/config-0.9.33.2/i386 b/toolchain/uClibc/config-0.9.33.2/i386
--- a/toolchain/uClibc/config-0.9.33.2/i386
+++ b/toolchain/uClibc/config-0.9.33.2/i386
@@ -19,3 +19,4 @@ CONFIG_486=y
 TARGET_ARCH="i386"
 TARGET_i386=y
 UCLIBC_HAS_FPU=y
+UCLIBC_HAS_BACKTRACE=y
